AzurePullRequests.Restore/Enums/RestoreType.cs

```diff
@@ -1,10 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
```

Comment:
Another user comments on a file

Reply:
Different user replies to another user's comment on file

Reply:
Original user replies to same thread with an attachment
![istockphoto-1398462038-612x612](https://github.com/user-attachments/assets/b3b855c8-5987-4990-9754-3ea20cec595d)